src/components/widgets/accounts-ratios/accounts-ratios.directive.coffee

```diff
@@ -33,6 +33,7 @@ module.controller('WidgetAccountsRatiosCtrl', ($scope, $q, $filter, $translate,
   $scope.intervalsCount = 0
   $scope.isPnl = false
   $scope.periodInfoContext = {}
+  $scope.allowXAxisLegend = true
```

Comment:
No need to initialise the property here as well as in the `w.initContext` initializer, please remove.